### PR TITLE
feature/auto-deploy

### PR DESCRIPTION
**Pull request by kamil-maslowski**
static files build for production

### CODE REVIEW

File: src/server/frontend/render.js [hunk 1]
```diff
@@ -90,17 +88,18 @@ async function renderPageAsync(store, renderProps, req) {
...
 }
 
-function getAppHtml(store, renderProps) {
-  return ReactDOMServer.renderToString(
-    <Provider store={store}>
-      <RoutingContext {...renderProps} />
-    </Provider>
-  );
-}
+// function getAppHtml(store, renderProps) {
```
Comment (burczu) [inline src/server/frontend/render.js]: remove commented code if it's working ;)

File: src/server/frontend/render.js [hunk 2]
```diff
@@ -113,19 +112,19 @@ async function getAppAssetFilenamesCachedAsync() {
...
-  // Note how clientState is serialized. JSON.stringify is anti-pattern.
-  // https://github.com/yahoo/serialize-javascript#user-content-automatic-escaping-of-html-characters
-  return scriptHtml + `
-    <script>
-      window.__INITIAL_STATE__ = ${serialize(clientState)};
-    </script>
-    <script src="${appScriptSrc}"></script>
-  `;
-}
+// function getScriptHtml(clientState, headers, hostname, appJsFilename) {
```
Comment (burczu) [inline src/server/frontend/render.js]: remove commented code if it's working ;)

File: src/server/frontend/render.js (hunk 1 above)
Comment (kamil-maslowski) [inline src/server/frontend/render.js]: Done...

File: src/server/frontend/render.js (hunk 2 above)
Comment (kamil-maslowski) [inline src/server/frontend/render.js]: ...and done.

**Pull request merged**